### ISSUE TITLE
Close Button Missing

### ISSUE THREAD

**Issue by phikal** [flathub/info.mumble.Mumble#16]
Hi,

I've installed the Flatpak on Fedora (w/ GNOME) to get a newer version of Mumble, and everything is working fine, except that the window buttons (close, and in my case maximize and minimize) are gone. Is this intentional?

**Comment by JBBgameich** [comment]
Fedora ships the GNOME wayland session by default, right?
Unfortunately this is related to the client side / server side decorations discussion. GNOME doesn't want to implement the wayland extension that makes the window manager capable of drawing window decorations in case the window (in this case, Mumble) doesn't provide its own.
Qt provides some fall back client side decoration, but maybe they are for some reason not functional in the KDE flatpak runtime.

In any case its not intentional, but I'm also not certain I can do much about it.

**Comment by phikal** [comment]
Yes, I'm using the Wayland, forgot to mention that. So your guess is that this is related to Qt and Wayland? I'll try to look into it then, in case I find anything, I'll post it here.

**Comment by JBBgameich** [comment]
Yes, I think it's related to Qt and Wayland on a desktop that doesn't support drawing window decorations.
It appears to be a common issue on GNOME, see for example https://github.com/flathub/org.kde.kate/issues/5

**Comment by phikal** [comment]
This issue seems to be resolved by the 1.3.2 update.

**Comment by maweil** [comment]
I can confirm that the current release (1.4.230) does have a close button in the title bar while using GNOME on Wayland.

**Comment by lokok8** [comment]
On Ubuntu 21.10, Gnome, Wayland I have no title bar. Mumble version 1.4.230.
Any ideas why its not working for me?

**Comment by maweil** [comment]
There were some KDE platform library updates yesterday, since then, the title bar is gone on my system as well.

**Comment by maweil** [comment]
There was another KDE application platform update today on my machine. Since then, the title bar (including the close button) is visible and works fine again.
@lokok8 : Could you please try updating your system and check whether this solved it for you as well?

**Comment by lokok8** [comment]
@maweil
I can confirm that this update fixed the problem for me. Thanks for the update notice!
